model/src/main/java/org/mskcc/cmo/metadb/model/PatientToNormalSampleEntity.java

```diff
@@ -0,0 +1,41 @@
+package org.mskcc.cmo.metadb.model;
+
+import java.io.Serializable;
+import org.neo4j.ogm.annotation.EndNode;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.RelationshipEntity;
+import org.neo4j.ogm.annotation.StartNode;
+
+/**
+ *
+ * @author ochoaa
+ */
+@RelationshipEntity("PX_TO_NORMAL")
+public class PatientToNormalSampleEntity implements Serializable {
+    @Id @GeneratedValue
+    private Long id;
+    @StartNode
+    private PatientMetadata patient;
+    @EndNode
+    private NormalSampleEntity normalSampleMetadata;
```

Comment:
Reminder:  Will need to update the type here after changing the name of the normal sample manifest entity class.